_posts/2024-12-20-OpenSearch-Simplified-The-Power-of-Application-Based-Templates.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "OpenSearch simplified: The power of Application-Based Configuration (ABC) templates"
+authors:
+   - mgodwani
+   - kolchfa
+date: 2024-12-19
+categories: 
+    - technical-post
+meta_keywords: 
+meta_description: 
+---
```

Comment:
Please add the following meta: 

_meta_keywords:_ Application-Based Configuration templates, ABC templates, index templates, index configuration, OpenSearch performance optimization

_meta_description:_ Explore OpenSearch's Application-Based Configuration (ABC) templates simplifies index configuration, optimizes performance, and streamlines setup for various use cases including logs, metrics, and traces.